### PR TITLE
feat(data/ScopeClosures/ch1.js): Add questions to Scope and Closures Chapter 1

### PR DESCRIPTION
**Pull request by austintackaberry**
#1

### CODE REVIEW

File: src/data/ScopeClosures/ch1.js [hunk 1]
```diff
@@ -79,6 +79,181 @@ const Ch1Questions = [
...
+      {
+        text:
+          'LHS looks for the variable container whereas RHS looks for the value of the variable',
+        id: 3,
+      },
+    ],
+    correctAnswerId: 3,
+    moreInfoUrl:
+      'https://github.com/getify/You-Dont-Know-JS/blob/master/scope%20%26%20closures/ch1.md#compiler-speak',
+    explanation: `LHS stands for "left hand side", and RHS stands for "right hand side". For a = b, the RHS looks for the value of b, and the LHS looks for the location of a in memory, so that it can update its value.`,
```
Comment (nikrb) [inline src/data/ScopeClosures/ch1.js]: `the LHS looks for the location of a in memory`
had to read this a few times, maybe bold 'a' ?

Comment (austintackaberry) [reply src/data/ScopeClosures/ch1.js]: I could put `a` in quotes. We don't yet have the ability to style the questions.

**Comment by nikrb** [comment]
at the risk of making a mountain out of a molehill, quotes don't look right imho, how about putting 'variable' in front of 'a'? If you think it's ok let's go with quotes - I have approved.

**Comment by austintackaberry** [comment]
Yeah, I know it isn't ideal, but I'm personally not too concerned because we will be getting support for markdown soon, and we will be able to truly fix it then.

**Comment by radotreyes** [comment]
I'm okay with how it is now. With React-Markdown our app will `eventually` `start` `looking` `like` `this`, so imo no need to make major formatting changes.

**Pull request merged**